LayoutTests/accessibility/mac/alt-for-css-content.html

```diff
@@ -53,14 +47,14 @@
 <div id="test5" test5="ALTERNATIVE CONTENT TEST5" aria-expanded="test5">test5</div>
 <div id="test6" test6="ALTERNATIVE CONTENT TEST6" aria-expanded="test6">test6</div>
 
-<img id="image1" src="#" alt="This is the right text" style="width:100px; height:100px; -webkit-alt: 'this is the wrong text';">
```

Comment:
Should you leave this one? It's testing the no-op.

Reply:
I can remove it.